infra/gcp/ensure-prod-storage.sh

```diff
@@ -150,6 +154,9 @@ empower_group_to_fake_prod \
 empower_group_to_fake_prod \
     "${GCR_BACKUP_TEST_PRODBAK_PROJECT}" \
     "k8s-infra-staging-cip-test@kubernetes.io"
+empower_group_to_fake_prod \
```

Comment:
Don't you need to enable pub/sub API and perms ?

Reply:
Sorry I missed this comment.

> Don't you need to enable pub/sub API and perms ?

Yes --- but I don't know which ones exactly. I'll cross that bridge soon...